src/layouts/MemberReport.js

```diff
@@ -0,0 +1,207 @@
+            <Typography className="member-report__info-label">
+              Date of Birth:&nbsp;
+            </Typography>
+            { memberInfo.dob || 'N/A' }
+          </Box>
+          <Box className="member-report__info-field">
+            <Typography className="member-report__info-label">
+              Age:&nbsp;
+            </Typography>
+            { memberInfo.dob ? getAge(memberInfo.dob) : 'N/A' }
+          </Box>
+          <Box className="member-report__info-field">
+            <Typography className="member-report__info-label">
+              Gender:&nbsp;
+            </Typography>
+            { memberInfo.gender || 'N/A' }
+          </Box>
+          <Box className="member-report__info-field">
+            <Typography className="member-report__info-label">
+              Coverage Status:&nbsp;
+            </Typography>
+            <Typography className={`member-report__coverage member-report__coverage--${coverage?.status.value || 'inactive'}`}>
+              { coverage?.status.value || 'inactive' }
+            </Typography>
+          </Box>
+          <Box className="member-report__info-field">
+            <Typography className="member-report__info-label">
+              Participation Period:&nbsp;
+            </Typography>
+            { coverage ? `${getDatestamp(new Date(coverage.period.start.value))} - ${
+              getDatestamp(new Date(coverage.period.end.value))}` : 'N/A' }
+          </Box>
+        </Grid>
+        {memberInfo.coverage && memberInfo.coverage.map((insurance) => (
+          <Grid key={`insurance-card-${insurance.id.value}`} className="member-report__patient-card">
+            <Box className="member-report__info-field">
+              <Typography className="member-report__info-label">
+                Policy ID:&nbsp;
+              </Typography>
+              { insurance.id.value }
+            </Box>
+            <Box className="member-report__info-field">
+              <Typography className="member-report__info-label">
+                Payor/Provider:&nbsp;
+              </Typography>
+              { insurance.payor[0]?.reference.value || 'N/A' }
+            </Box>
+            <Box className="member-report__info-field">
+              <Typography className="member-report__info-label">
+                Plan:&nbsp;
+              </Typography>
+              N/A
+            </Box>
+            <Box className="member-report__info-field">
+              <Typography className="member-report__info-label">
+                Dependents:&nbsp;
+              </Typography>
+              N/A
+            </Box>
+            <Box className="member-report__info-field">
+              <Typography className="member-report__info-label">
+                Relationship:&nbsp;
+              </Typography>
+              <Typography className="member-report__relationship-label">
+                {insurance.relationship.coding[0]?.code.value}
```

Comment:
I used data created from the test deck to test the member page. Gave me a null pointer exception. I just adding another `?` fixes it

`{insurance.relationship?.coding[0]?.code.value}`